aspnetcore/blazor/security/webassembly/azure-active-directory-groups-and-roles.md

```diff
@@ -382,6 +555,40 @@ For example:
 },
 ```
 
+::: moniker range=">= aspnetcore-5.0"
+
+> [!NOTE]
+> If the tenant publisher domain isn't verified, the server API scope for user/client access uses an `https://`-based URI. In this scenario, the server API app requires `Audience` configuration in the `appsettings.json` file:
+>
+> ```json
+> {
+>   "AzureAd": {
+>     ...
+>
+>     "Audience": "https://{TENANT}.onmicrosoft.com/{SERVER API APP CLIENT ID OR CUSTOM VALUE}"
+>   }
+> }
+>
+> In the preceding configuration, the end of the `Audience` value does **not** include the default scope `/{DEFAULT SCOPE}`.
```

Comment:
Maybe move this above the example for cohesion. And structure the example so that it shows the sample value `https://contoso.onmicrosoft.com/41451fa7-82d9-4673-8fa5-69eff5a761fd` and highlights which parts are the tenant and app id in a table below?